Microsoft.DotNet.Interactive.CSharp/CSharpKernelExtensions.cs

```diff
@@ -84,8 +59,51 @@ public static CSharpKernel UseNugetDirective(this CSharpKernel kernel)
             kernel.AddDirective(restore);
 
             return kernel;
+        }
+
+        private static Command i(PackageRestoreContext restoreContext)
+        {
+            var iDirective = new Command("#i")
+            {
+                new Argument<string>("source")
+            };
+            iDirective.Handler = CommandHandler.Create<string, KernelInvocationContext>((source, context) =>
+            {
+                restoreContext.AddRestoreSource(source.Replace("nuget:", ""));
```

Comment:
I think here we call PackageRestoreContext a NuGetPacakgeRestoreContext. if we want to be pluggable we should rename this type to express that we are talking about Nuget sources and NuGet packages.

Reply:
I don't think we know what generalization looks like until we try to build a second implementation so it's fine for "package" to mean "NuGet package" for now. Much of the code in `PackageRestoreContext` would be the same for other package managers.